internal/db/update.go

```diff
@@ -56,6 +56,7 @@ CREATE TABLE instance_overrides (
 CREATE TABLE networks (
     id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
     name VARCHAR(255) NOT NULL,
+	  location TEXT NOT NULL,
```

Comment:
We are using Migration Manager in production now, so changes like this must be done through a schema update function now.

Reply:
Fixed. We'll likely need some additional validation soon to prevent Migration Manager from fully starting up until the initial source sync updates the database with values for the new columns. 

We'll probably also need to have a dedicated workaround for associating pre-existing networks with a source once we add a column for that as well.

Reply:
Oh, forgot to push :)